### ISSUE TITLE
Text field grows on focus out in safari

### ISSUE THREAD

**Issue by b-graves** [formaat-design/reshaped#178]
See gif, in safari, the text field grows each time the focus is lost

![zFMEADUz9J](https://github.com/formaat-design/reshaped/assets/56936483/16e23f89-bd41-493d-9b53-aa520e7d4e0a)

code as follows:

```
<View direction="row" align="end" padding={3}>
  <View.Item>
    <TextField
      name="search"
      placeholder="Search projects"
      onChange={(e) => setGlobalFilter(e.value)}
      value={globalFilter}
    />
  </View.Item>
  <View.Item gapBefore="auto">
    <Button color="primary" onClick={onCreateNew}>
      New Project
    </Button>
  </View.Item>
</View>
```

**Comment by blvdmitry** [comment]
Thanks for heads-up, I'll take a look today and will release a patch once it's ready

**Comment by blvdmitry** [comment]
Should be fixed in 2.4.3
https://codesandbox.io/s/nervous-murdock-kxgrnj?file=/src/App.tsx
